src/libs/geolocation.js

```diff
@@ -10,20 +11,24 @@ const geolocationPermissions = {
 let hasPermissionModalOpenedOnce = false;
 
 export async function showGeolocationModalIfNeeded() {
+  if (!isMobileDevice()) {
+    return false;
+  }
+
+  // Some browsers (Safari, etc) do not implement Permissions API
   if (!window.navigator.permissions) {
-    // Some browsers (Safari, etc) do not implement Permissions API
-    return;
+    return false;
```

Comment:
If I understand correctly, this means we'll never request the position on Safari.

Reply:
Yes, in the current implementation, Safari won't see the Modal and won't request position when opening Directions panel.
We'll see after pushing in prod if the modal choice should rather be saved in localStorage (in that case, we can be sure it only appears once and can be enabled on Safari too without depending on the Permissions api)